statsmodels/base/model.py

```diff
@@ -2213,13 +2213,17 @@ def bootstrap(self, nrep=100, method='nm', disp=0, store=1):
         """
         results = []
         print(self.model.__class__)
-        hascloneattr = True if hasattr(self, 'cloneattr') else False
+        hascloneattr = True if hasattr(self.model, 'cloneattr') else False
```

Comment:
Is cloneattr documented somewhere?

Reply:
No, it's not advertised.
discussion in the original PR